### PR TITLE
chore(update): Updating Channel to trigger an Operator Update

### PR DESCRIPTION
**Pull request by Fortune-Ndlovu**
## Which issue(s) does this PR fix or relate to

https://issues.redhat.com/browse/RHIDP-4687 

## PR acceptance criteria

- [ ] Tests
- [ ] Documentation
- [ ] If the bundle manifests have been updated, make sure to review the [`rhdh-operator.clusterserviceversion.yaml`](../.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml) file accordingly

### CODE REVIEW

File: .rhdh/bundle/manifests/rhdh-operator.csv.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ metadata:
     features.operators.openshift.io/token-auth-gcp: "false"
     repository: https://gitlab.cee.redhat.com/rhidp/rhdh/
     support: Red Hat
-    skipRange: '>=1.0.0 <1.3.2'
+    skipRange: '>=1.2.0 <1.3.2'
```
Comment (nickboldt) [inline .rhdh/bundle/manifests/rhdh-operator.csv.yaml:45]: while valid, why do we need to change this?

**Review by nickboldt — APPROVED** [review]

File: .rhdh/bundle/manifests/rhdh-operator.csv.yaml (hunk 1 above)
Comment (rm3l) [reply .rhdh/bundle/manifests/rhdh-operator.csv.yaml:45]: Starting from 1.0.0 means a direct update from, say, 1.1.0 to 1.3.2, no? Could this be an issue, knowing the possible breaking changes that might happen across minor versions of the operand? Would it make more sense to allow some kind of "progressive" minor upgrades?

Comment (Fortune-Ndlovu) [reply .rhdh/bundle/manifests/rhdh-operator.csv.yaml:45]: Yes, direct updates from earlier versions like 1.1.0 to 1.3.2 could introduce issues. To address this, I am maintaining the replaces field 1.2.5 -> 1.3.2 and ensuring the skipRange limits upgrades to versions within the 1.2.x series.

File: .rhdh/bundle/manifests/rhdh-operator.csv.yaml [hunk 2]
```diff
@@ -357,4 +357,4 @@ spec:
     name: Red Hat Inc.
     url: https://www.redhat.com/
   version: 1.3.2
-  replaces: rhdh-operator.v1.2.4
+  replaces: rhdh-operator.v1.2.5
```
Comment (nickboldt) [inline .rhdh/bundle/manifests/rhdh-operator.csv.yaml:360]: Need to change this to the version of 1.2 that was released on the fast channel - 1.2.0 or something

Comment (nickboldt) [reply .rhdh/bundle/manifests/rhdh-operator.csv.yaml:360]:
```suggestion
  replaces: rhdh-operator.v1.2.3
```

**Review by nickboldt — APPROVED** [review]

**Pull request merged**